smartsim/_core/_install/builder.py

```diff
@@ -369,12 +371,32 @@ def build_from_git(self, git_url, branch, device):
             "clone",
             "--recursive",
             git_url,
-            "--branch",
-            branch,
-            "--depth=1",
-            "RedisAI",
         ]
+        # Circumvent a bad `get_deps.sh` script from RAI on 1.2.7 with ONNX
+        # TODO: Look for a better way to do this or wait for RAI patch
+        if sys.platform == "darwin" and branch == "v1.2.7" and self.onnx:
+            # Clone RAI in an insane way
```

Comment:
This is just grabbing a particular commit, right? Let's not say "insane"

Reply:
Good call! Probably a bit tasteless on my end as well